PurchasesTests/Mocks/MockETagManager.swift

```diff
@@ -27,11 +28,19 @@ class MockETagManager: ETagManager {
             return stubbedETagHeaderResult
         }
     }
-    
+
+    struct InvokedHTTPResultFromCacheOrBackendParams {
```

Comment:
This can be `private`.

Reply:
sure!

Reply:
Not really. This struct is used to encapsulate the parameters of the method and we have some vars of this type

https://github.com/RevenueCat/purchases-ios/blob/79449bf53fc8113c0c479f0841a17e4e940d190e/PurchasesTests/Mocks/MockHTTPClient.swift#L22

Am I missing something? 🤔

Reply:
Oooh I see that you put this inside `MockETagManager`. Never mind then.

Reply:
yeah, it's a nested type 👍🏼